### ISSUE TITLE
Very poor result

### ISSUE THREAD

**Issue by rose-jinyang** [josephch405/cs348k-zoom-matting#6]
Hello
How are you?
Thanks for contributing this paper and project.
I tested this project with several images and web camera.
The result is very bad.

![image](https://user-images.githubusercontent.com/47862419/93634722-a5c26000-fa23-11ea-836d-69c12fb4d732.png)

![image](https://user-images.githubusercontent.com/47862419/93634987-14072280-fa24-11ea-832f-097973e73976.png)

The right images are the mask images resulted from the pre-trained jitnet model by u.
I tested with several input sizes (128x128, 224x224, 512x512).
The paper said that this method outperforms U2-Net method in accuracy and speed.
![image](https://user-images.githubusercontent.com/47862419/93635506-ff775a00-fa24-11ea-87fc-3c5108d469a7.png)

But these results are so bad.
U2-Net model is slow but very accuracy.
How can I understand this?

**Comment by josephch405** [comment]
Hi -
We didn't train the starter JIT-Net model for very long and so you'd have to run this on a video input in order for the distillation effects to manifest. Therefore, as expected, single shot inference on just the image itself does quite poorly. You would need to train the small JIT-Net model on a better corpus of facial images if you want a better pretrained model (we didn't get around to this in time).
